features/patroni_api.feature

```diff
@@ -14,9 +14,9 @@ Scenario: check API requests on a stand-alone server
 	Then I receive a response code 200
 	When I issue a GET request to http://127.0.0.1:8008/replica
 	Then I receive a response code 503
-	When I run patronictl.py reinit batman postgres0 --force
-	Then I receive a response returncode 0
-	And I receive a response output "Failed: reinitialize for member postgres0, status code=503, (I am the leader, can not reinitialize)"
+	When I issue a POST request to http://127.0.0.1:8008/reinitialize with {"force": true}
+	Then I receive a response code 503
+	And I receive a response text I am the leader, can not reinitialize
```

Comment:
Yes, it was necessary.
I did some refactoring of patronictl while introducing `--group` option and now it doesn't allow to call `POST /reinitialize` if member is a leader.
Since this behave test is about REST API I decided to call it directly, without patronictl.